flake.nix

```diff
@@ -124,6 +125,43 @@
                   sops-nix.homeManagerModules.sops
                 ];
               }
+              {
+                boot.loader.systemd-boot.enable = true;
+                boot.loader.efi.canTouchEfiVariables = true;
```

Comment:
I checked on my personal laptop that for kexec images there are unnecessary, since it passes through the bootloader. It's possible to pass `boot.loader.grub.enable = false;` to disable any bootloader on such hosts, which brings them to uniformity with RPi4.

I think for now, we can keep these like these, and create a separate PR in which we also diff each hosts which has these to verify that neither grub or systemd-boot are present by grepping the graph or tree output of the nix diff hash.